docker/dockerfiles/Dockerfile.onnx.jetson.6.0.0

```diff
@@ -0,0 +1,80 @@
+FROM roboflow/l4t-ml:r36.3.0
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV LANG=en_US.UTF-8
+
+# Install dependencies
+RUN apt-get update -y && \
+    apt-get install -y --no-install-recommends \
+    lshw \
+    git \
+    gfortran \
+    build-essential \
+    libatlas-base-dev \
+    libsm6 \
+    libxext6 \
+    wget \
+    gdal-bin \
+    libgdal-dev && \
+    rm -rf /var/lib/apt/lists/*
+
+# Copy all requirements files
+COPY requirements/ ./requirements/
+
+# Remove specific packages from requirements files
+RUN sed -i '/^opencv-python/d;/^onnxruntime/d;/^opencv-python-contrib/d' requirements/*.txt
```

Comment:
@alexnorell with your changes do you think we still need this line?

Reply:
Probably not. We can remove it and see if there are any issues.

Reply:
Update: talked to Alex & we're going to leave this because it's probably why adding onnxruntime-gpu to the new jetson requirements doesn't clobber the built version from the base image